src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+const CHECK_STATUS = 'CHECK_STATUS';
+
```

Comment:
- [ ] Great job defining the `CHECK_STATUS` as action type 👏🏻  but the value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`